docs/content/docs/user-docs/install.md

```diff
@@ -59,19 +59,28 @@ helm pull oci://$CHART_REPO --version $RELEASE_VERSION -d $CHART_EXPORT_PATH
 tar xvf $CHART_EXPORT_PATH/$CHART_PACKAGE -C $CHART_EXPORT_PATH
 ```
 
+{{% hint type="info" title="s3-controller version" %}}
+Above commands will download the latest version of `s3-controller`. To select a
+different version, change `RELEASE_VERSION` variable and execute the commands again.
```

Comment:
:+1: 

I actually think that putting the `git` command from above **here** in this hint would be a better idea. Basically, something like "if you have Git on your system, you can find the latest release tag using this Bash snippet..."